lib/plugins/gulp-silent-sass.js

```diff
@@ -23,7 +27,7 @@ function gulpSilentSass(config) {
 	}
 
 	// creating a stream through which each file will pass
-	var stream = through.obj(function(file, enc, cb) {
+	stream = through.obj(function(file, enc, cb) {
```

Comment:
I'd be careful of using module scope to share objects like this, particularly when there is a chance that 'stream' could be undefined.  In this narrow example it might be better to create stream in gulpSilentSass and pass it as a parameter?